packages/react-components/react-drawer/src/shared/useDrawerBaseStyles.styles.ts

```diff
@@ -53,6 +53,12 @@ export const useDrawerBaseStyles = makeStyles({
     right: 0,
     left: 'auto',
   },
+  bottom: {
+    bottom: 0,
+    top: 'auto',
+    height: 'calc(100vh - 188px)',
```

Comment:
@marcosmoura  
This height value is what our design spec currently uses, we don't have a set of sizes for height. Should we create a set of sizes for bottom-to-up drawer? 

And the current size prop does not work with position "bottom". Should I create a separate component or keep the same here but indicate that the current size prop does not work with bottom position in the doc?

Reply:
Thanks for suggesting! Myself don't have a set of size for bottom, I use the same set of numbers other than changing the full size to use vh. 

The change: https://github.com/microsoft/fluentui/pull/29137/files#diff-826a5679f4ac6e35ae477b749c6eb88fe65512fe44289db8149af098da92912eR78